tensorflow/python/framework/errors_impl.py

```diff
@@ -47,11 +48,17 @@ def __init__(self, node_def, op, message, error_code):
       error_code: The `error_codes_pb2.Code` describing the error.
     """
     super(OpError, self).__init__()
-    self._message = message
     self._node_def = node_def
     self._op = op
+    self._message = message
     self._error_code = error_code
 
+  def __reduce__(self):
```

Comment:
Alternatively, I can explicitly implement `__reduce__` in each of the subclasses, wdyt?